terminus/tests/road_node_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from models.road_node import RoadNode
+
+from geometry.bounding_box import BoundingBox
+
+from geometry.point import Point
+
+
+class RoadNodeTest(unittest.TestCase):
+
+    def test_road_node_bounding_box(self):
```

Comment:
Why would that happen? I suppose it would return the same box as the one with its absolute value. I can write an error message if necessary, but I think that is just not supposed to happen.